content/news/playtest-20160424.md

```diff
@@ -0,0 +1,69 @@
+We are happy to present to you the second playtest in this release cycle!
+
+Many users &ndash; indeed more than we anticipated &ndash; have tried out the first playtest and gave us a lot of feedback. Almost inevitable, a number of bugs have also been reported (thanks to all the testers who took their time to do so!), many of them quite hard to track down.
+
+We have now fixed all of those issues, including:
+
+* Crashes when opening the in-game menu on some maps (Fort Lonestar, for example).
+* Broken left click orders.
+* Way too many infantry units coming out of destroyed buildings.
+* Submarines not being targetable.
+* Stealth units being targetable even when stealthed under some circumstances.
+
+Additionally, the Red Alert mod has seen changes to its map pool: a few lower quality maps have been removed, while some new tournament-grade maps have been added.
+
+As usual, the complete list of changes can be seen [on our wiki](http://github.com/OpenRA/OpenRA/wiki/Changelog/HASHHASHASH).
+
+We promised in the last news post to detail the new map format and other map-related changes here. Skip to below the break for the full scoop.
+
+Installers for all supported operating systems are available from [the download page](/downloads). Of course we are still interested to hear about any bugs and issues you come across, so please don't hesitate to report them [on our bug tracker](http://bugs.openra.net), in the OpenRA [IRC channel](/community/), or here in the comments.
+
+The following note is copied from the previous news post, but we are going to repeat it here for accessibility.<br/>
+As many other open-source projects, we don't have any clear idea how much our software is actually used. There is a huge discrepancy between the number of downloads (about 50,000 currently for the latest release) and the number of online players (60-75 players concurrently at peak times). In order to get a better idea of this, and also to help with steering the future direction of the project, we decided to add an anonymous, opt-out data gathering facility to OpenRA. When enabled, the game will send us your operating system and .NET framework version, supported OpenGL version of your graphics card, your language settings, as well as the current mod. As already mentioned, this data is sent to us anonymously, using a randomly generated ID. If you are still feeling uncomfortable with this, you can disable this functionality by unchecking the "Send System Information" checkbox in the advanced settings dialog, or by starting the game using the `Debug.SendSystemInformation=false` argument.
```

Comment:
I think we should reword this, with reference to the comment thread from the last news post.  We can reduce the scope of the "why" and make the "what" more obvious for people who don't read properly.